test/common/test_context.go

```diff
@@ -94,7 +94,6 @@ type TestContext struct {
 	SM          *httpexpect.Expect
 	SMWithOAuth *httpexpect.Expect
 	SMWithBasic *httpexpect.Expect
-	SMServer    *httptest.Server
```

Comment:
Why remove SMServer?

Reply:
The reference to the httptest.Server is lost during the construction of testContext, than no one is responsible to close it.

Reply:
It is created on line 88 (https://github.com/Peripli/service-manager/blob/e4d08df3954ad01b38f837fa7854ec0f61c1e4d4/test/common/test_context.go#L88) and never closed.